website/src/content/docs/blog/biome-wins-prettier-challenge.mdx

```diff
@@ -0,0 +1,145 @@
+authorAvatar: https://avatars.githubusercontent.com/u/602478?v=4
+authorUrl: https://twitter.com/ematipico
+coauthorName: Biome Core Team, Biome Maintainers
+coauthorAvatar: /img/circle-indent-logo.svg
+pubDate: 2023-11-27
+layout: ../../../layouts/Blog.astro
+coverImage: ../../src/assets/prettier-challenge.png
+coverImageAltText: The Prettier challenge banner, with the Biome logo over it
+socialImage: ../../src/assets/prettier-challenge.png
+---
+
+With the release of Biome **`v1.4.0`**, we claim the bounty of the [Prettier challenge](https://console.algora.io/challenges/prettier)!
+
+With `v1.4.0`, you'll get a better formatter experience, more formatting options, new VSCode features, new sponsors and more!
+
+You can upgrade Biome by running the following command:
+
+```bash
+npm install --save-dev --save-exact @biomejs/biome@1.4.0
+pnpm update --save-exact @biomejs/biome@1.4.0
+yarn upgrade --exact @biomejs/biome@1.4.0
+```
+
+##  Better formatter
+
+Biome formatter has now **over 95% in terms of compatibility** against [Prettier](https://prettier.io/)!
+
+Merit of challenge that was launched by [Christopher Chedeau](https://twitter.com/Vjeux/status/1722733472522142022), one of the Prettier's creators.
+
+The challenge attracted the attention of many people, and some of them decided to contribute to Biome to claim part of the bounty. I did see something amazing: contributors had an amazing coordination, they took ownership of the tasks and delivered the solution in a matter of hours.
+
+I believe the main factors that made this possible are three:
+1. Money. It's a fact, and it's completely fine if someone decides to contribute only for earning a small stipend out of it.
+2. Communication. We used [GitHub](https://github.com/biomejs/biome/issues/720) as only medium of coordination. We provided information, instructions and help on how to deliver.
+3. Infrastructure. Biome relies on a solid testing infrastructure, built by [previous](https://github.com/MichaReiser) Rome Tools [employees](https://github.com/ematipico) and [contributors](https://github.com/IWANABETHATGUY/). It's able to catch every idempotency bug, provide granular diffs and warn the user if the emitted output is the different from the one emitted by Prettier.
+
+Before the challenge, Biome had roughly a compatibility rate of 85%, based on our internal metrics (JavaScript, TypeScript and JSX, on options parity). Even though 85% might seem high,
+the impact of a low number such as 15% on big code bases is huge, and people might feel intimidated by so many changes, causing early adopters to receive frictions when bring Biome to their team. A member of our community shared some insights:
+
+> As a great example of how much even just that last 5% has improved things for large codebases (and specifically with `bracketSpacing` and now `bracketSameLine` implemented) i ran it one project in our monorepo (it's a debug build on a busy machine, that's why it took 13 seconds).
```

Comment:
We could omit some details. The `13s in debug build` can provide a wrong idea of Biome perf?
```suggestion
> As a great example of how much even just that last 5% has improved things for large codebases (and specifically with `bracketSpacing` and now `bracketSameLine` implemented) i ran it one project in our monorepo [...].
```